utils/verify_intel_gaudi/roles/verify_intel_gaudi/tasks/hlqual_gaudi3_validation.yml

```diff
@@ -147,49 +147,6 @@
   failed_when: "'FAILED' in serdes_base_dirbw_test_result.stdout"
   changed_when: true
 
-- name: Unload habanalabs kernel module
-  community.general.modprobe:
-    name: habanalabs
-    state: absent
-
-- name: Load habanalabs kernel module with timeout_locked param
-  community.general.modprobe:
-    name: habanalabs
-    state: present
-    params: 'timeout_locked=0'
-
-- name: Bring DOWN all Gaudi3 NICs
-  ansible.builtin.shell: |
-    set -o pipefail
```

Comment:
this code is not removed in hlqual_gaudi2_valiadtion.yml. Should we remove from there as well?